frontend/components/hypercert-create.tsx

```diff
@@ -295,14 +293,11 @@ export interface HypercertCreateFormProps {
 export function HypercertCreateForm(props: HypercertCreateFormProps) {
   const { className, children } = props;
   const { address } = useAccountLowerCase();
-  const { chain } = useNetwork();
   const { push } = useRouter();
   const { hideModal } = useContractModal();
   const confetti = useConfetti();
-  const { client } = useHypercertClient();
-  const { data: balance, isLoading: balanceLoading } = useBalance({
-    address: address as `0x${string}`,
-  });
+
+  const { checkWriteable } = useCheckWriteable();
```

Comment:
ya same here, why not just read `writeable`?

Reply:
Updated in current commit